addons/sale_management/models/sale_order.py

```diff
@@ -21,6 +21,9 @@ class SaleOrder(models.Model):
         comodel_name='sale.order.option', inverse_name='order_id',
         string="Optional Products Lines",
         copy=True)
+    journal_id = fields.Many2one(
+        store=True, readonly=False,
+        compute="_compute_journal_id", precompute=True)
```

Comment:
Would be better for the global architecture to directly make it compute in the `sale` module (even if it's an empty compute, setting the value to False/None)